data/mods/Aftershock/items/corpses.json

```
@@ -287,7 +256,7 @@
     "symbol": ",",
     "color": "white",
     "looks_like": "broken_tribot",
-    "name": "broken medical bot",
+    "name": "broken surgical droid",
     "category": "other",
     "description": "A broken medical robot, now crumpled and inert.  Could be stripped down for parts.",
```

```suggestion
    "description": "A broken surgical robot, now crumpled and inert.  Could be stripped down for parts.",
```
Given the name-change.